src/test/java/org/eolang/speco/MainTest.java

```diff
@@ -33,11 +42,81 @@
  */
 public final class MainTest {
 
+    /**
+     * Relative path to the directory with tests.
+     */
+    private final Path tests = Path.of("./src/test/resources");
```

Comment:
Please use File.separatorChar or its analogues instead of '/' and '\' symbols. File separators are platform-dependent and using '/' explicitly may lead to test failures on other operating systems (this might be the reason why tests for UNIX fail btw). [Take a look](https://www.digitalocean.com/community/tutorials/java-file-separator-separatorchar-pathseparator-pathseparatorchar)

Reply:
You can also use:
```java
Path.of(".").resolve("src").resolve("test").resolve("resources");
```

Reply:
@OlesiaSub @volodya-lombrozo `Path.of` supports varargs, so it can be reduced to:
```java
Path.of(".", "src", "test", "resources");
```